sphinx_scylladb_theme/layout.html

```diff
@@ -169,16 +169,13 @@
         <div class="admonition caution">
           <p class="admonition-title">Caution</p>
           <p>
-            {% if current_version.name == theme_default_branch %} You are
-            reading an unstable version of this documentation. {% else %} You
-            are not reading the most current version of the documentation. {%
-            endif %} If you want up-to-date information, please have a look at
-
+            You are not reading the latest stable version of this documentation.
+            If you want up-to-date information, please have a look at
```

Comment:
why is the condition removed?

Reply:
@tzach @dgarcia360 And what was the reason the condition was removed? (it relates to something I'm currently checking)